src/ui/shared/Search.tsx

```diff
@@ -59,7 +59,7 @@ export default function Search({ onSearch, onClear }: SearchProps) {
               value={query}
               onChange={handleInputChange}
               placeholder="Search..."
-              className="w-full bg-twd-background px-3 py-2 rounded-md border border-gray-300"
+              className="w-full bg-twd-background px-3 py-2 rounded-md border border-gray-300 focus:border-none focus:outline-none focus:ring-2 focus:ring-twd-primary-purple"
```

Comment:
Same same

Reply:
without it the browser/device overrides the styling, so our purple outline isn't visible. As far as I'm aware, the fact that we're removing it but adding our own custom outline with focus:ring should mean that this element meets accessibility requirements